py/bitbox02/bitbox02/bitbox02/bootloader.py

```diff
@@ -94,6 +104,17 @@ def versions(self) -> typing.Tuple[int, int]:
         firmware_v, signing_pubkeys_v = struct.unpack("<II", response[:8])
         return firmware_v, signing_pubkeys_v
 
+    def hardware(self) -> Hardware:
+        """
+        Returns (hardware variant).
+        """
+        # Previous bootloader versions does not support the call and have ATECC SC.
+        if self.version > "1.0.6":
+            response = self._query(b"W")
+            if response[:2] == b"\x01":
```

Comment:
how can two bytes equal one byte? :grin:

Reply:
Shoot, how did I miss this? :see_no_evil:

Reply:
Lol, it worked by coincidence because the `query` call already strips the first 2 bytes, so `response[:2]` was actually 1 byte long

Reply:
Fixed :+1: